sphinx/source/concepts.rst

```diff
@@ -12,7 +12,7 @@ Network and Nodes
 
 A CCF network consists of several nodes, each running on top of a Trusted Execution Environment (:term:`TEE`), or enclave, such as :term:`SGX`. As such, a CCF network is decentralised and highly-available.
 
-Nodes are run and maintained by :term:`operators`. However, nodes must be trusted by the consortium of members before joining a CCF network.
+Nodes are run and maintained by :term:`operators`. However, nodes must be trusted by the consortium of members before participating in a CCF network.
```

Comment:
I think that `joining` is fine. `Participating` involves that they are active in the consensus but we could imagine a scenario where nodes join first and only participate once they have caught up.

Reply:
As it is it implies that there is a step that needs to happen first by the members before a node can join the network, where as we first start up a node with `join` and then ask to be accepted by the members. Maybe not `participating` but something else?